src/prometheus/deploy/alerting/pai-services.rules

```diff
@@ -34,8 +34,8 @@ groups:
         annotations:
           summary: "{{$labels.name}} in {{$labels.host_ip}} not ready detected"
 
-      - alert: PaiServiceNotUp
-        expr: up != 1
+      - alert: PaiServiceWatchDogNotUp
```

Comment:
Why change this?

Reply:
> Why change this?

This alert has overlap with PaiServieNotRunning|PaiServiceNotReady. Renamed to `WatchdogNotUp`.